testgrid/config/config.yaml

```diff
@@ -1715,6 +1735,26 @@ dashboards:
     test_group_name: ci-kubernetes-e2e-gci-gce-serial-release-1.6
   - name: gci-gce-slow-1.6
     test_group_name: ci-kubernetes-e2e-gci-gce-slow-release-1.6
+  - name: gci-gke-ingress-1.6
+    test_group_name: ci-kubernetes-e2e-gci-gke-ingress-release-1.6
+  - name: gci-gke-reboot-1.6
+    test_group_name: ci-kubernetes-e2e-gci-gke-reboot-release-1.6
+  - name: gci-gke-1.6
+    test_group_name: ci-kubernetes-e2e-gci-gke-release-1.6
+  - name: gci-gke-serial-1.6
+    test_group_name: ci-kubernetes-e2e-gci-gke-serial-release-1.6
+  - name: gci-gke-slow-1.6
+    test_group_name: ci-kubernetes-e2e-gci-gke-slow-release-1.6
+  - name: gke-alpha-features-1.6
+    test_group_name: ci-kubernetes-e2e-gke-alpha-features-release-1.6
+  - name: gke-reboot-1.6
+    test_group_name: ci-kubernetes-e2e-gke-reboot-release-1.6
+  - name: gke-1.6
+    test_group_name: ci-kubernetes-e2e-gke-release-1.6
+  - name: gke-serial-1.6
+    test_group_name: ci-kubernetes-e2e-gke-serial-release-1.6
+  - name: gke-slow-1.6
+    test_group_name: ci-kubernetes-e2e-gke-slow-release-1.6
 
 - name: release-1.6-blocking
```

Comment:
this list is pretty arbitrary right now (it sorta matches https://github.com/kubernetes/release/blob/e5e6416b02d117a50494986a6fd8386fae5b21d5/lib/releaselib.sh#L100-L117).

Someone should formalize this.